docs/developer-guide.md

```diff
@@ -0,0 +1,84 @@
+# Pulsar Manager Developer's Guide
+
+Pulsar Manager is a web-based GUI management and monitoring tool that helps administrators and users manage and monitor tenants, namespaces, topics, subscriptions, brokers, clusters, and so on, and supports dynamic configuration of multiple environments.
+
+![architecture](img/pulsar-manager-architecture.png)
+
+The figure above illustrates how does Pulsar Manager work and how it manages multiple Pulsar environments. The Pulsar Manager comprises two parts: frontend and backend.
+
+## Backend
+
+Development tools used in the backend are listed below:
+
+* Spring-boot 
+* Mybatis
+* PageHelper
+* Swagger2
+* Zuul
+
+The code structure of the backend is shown below.
+
+![backend-code](img/pulsar-manager-backend-code.png)
+
+The backend configuration file is [here](https://github.com/streamnative/pulsar-manager/blob/master/src/main/resources/application.properties).
```

Comment:
```suggestion
* The backend configuration file is [here](https://github.com/streamnative/pulsar-manager/blob/master/src/main/resources/application.properties).
```

Reply:
Currently, it looks like:  

![image](https://user-images.githubusercontent.com/50226895/64432668-9231a900-d0ef-11e9-9049-6546b07207fd.png)